scripts/audio-format-converter/script.py

```diff
@@ -0,0 +1,64 @@
+# Script to Convert audio files from one format to another
+import os
+from glob import glob
+import pydub
+import mutagen
+import keyboard
+
+# Directory containing the audio File
+# Extension is according the the file type before conversion.
+song_dir = "./*.mp3"
```

Comment:
Can we allow the user to input the directory over here instead of editing the script? Make the necessary changes in your readme as well once you are done adding the input field.

Reply:
okay, I'll make it work, such that the audio file to be converted is needed in the working dir.